doc/user/content/ingest-data/ingest-troubleshooting.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Troubleshooting ingest"
+description: ""
+menu:
+  main:
+    name: "Troubleshooting ingest"
+    identifier: ingest-troubleshooting
+    parent: ingest-data
+    weight: 40
+---
+
+If you're looking for somewhere to start for troubleshooting slow or unresponsive queries, start at the [troubleshooting](troubleshooting) page.
+
+<!-- Copied over from the old manage/troubleshooting guide -->
+## Why isn't my source ingesting data?
+First check the status of your source in the Materialize console by going to https://console.materialize.com/, clicking the "Sources" tab in the nav bar, and clicking into the source.
+
+
+You can also query for this information via SQL in the [`mz_source_statuses`](/sql/system-catalog/mz_internal/#mz_source_statuses) table:
+
+```sql
+SELECT * FROM mz_internal.mz_source_statuses
+WHERE name = <SOURCE_NAME>;
+```
+
+If your source reports a status of `stalled` or `failed`, you likely have a
+configuration issue. The returned `error` field will provide details.
+
+If your source reports a status of `starting` for more than a few minutes,
+[contact support](/support).
+
+If your source reports a status of `running`, but you are not receiving data
+when you query the source, the source may still be ingesting its initial
+snapshot. See the next section.
+
+## Has my source ingested its initial snapshot?
```

Comment:
Let's add a sentence defining what the initial snapshot is and why it has to be committed atomically.

Reply:
Good suggestion. I added a small explainer.
I took content from https://materialize.com/docs/ingest-data/postgres-alloydb/, which is a great walkthru. @morsapaes maybe we can figure out how to fold this walkthru into the troubleshooting docs somehow and/or make it more generic?

Reply:
I'm hopeful that we can soon just point users to the console. This will be exposed as part of the source progress work, right?

Reply:
Ah yeah I think so. We're also already in private preview for query history in the console, so will add content for both of those soon. I just added a phase 2 to the troubleshooting docs tracking ticket for all the follow up items like query history in console, source progress in console, and incorporating all of chuck's queries.

Reply:
Oh, now that I'm going through it, I understand what you mean by "making it more generic". Took a stab at it:

> When you create a source, Materialize takes a snapshot of any existing data in
the upstream external system and ingests that snapshotted data before it starts
ingesting new data. To ensure correct and consistent results, this snapshot is
committed atomically at a specific timestamp. Because of that, you will not be
able to query your source (or, queries will return no data) until Materialize
has finished ingesting the initial snapshot.

Reply:
You don't need to work this in, @chaas! I started reviewing this and will just push a commit with this sort of edits. I'll ping you for one last look tomorrow, and 🤞 we're all set to merge then!